scripts/react_native_pods.rb

```diff
@@ -63,6 +63,23 @@ def use_flipper!(version = '~> 0.33.1')
   pod 'FlipperKit/SKIOSNetworkPlugin', version, :configuration => 'Debug'
   pod 'FlipperKit/FlipperKitUserDefaultsPlugin', version, :configuration => 'Debug'
   pod 'FlipperKit/FlipperKitReactPlugin', version, :configuration => 'Debug'
+
+  # List all transitive dependencies for FlipperKit pods
+  # to avoid them being linked in Release builds
+  pod 'Flipper', version, :configuration => 'Debug'
+  pod 'Flipper-DoubleConversion', '1.1.7', :configuration => 'Debug'
+  pod 'Flipper-Folly', '~> 2.1', :configuration => 'Debug'
+  pod 'Flipper-Glog', '0.3.6', :configuration => 'Debug'
+  pod 'Flipper-PeerTalk', '~> 0.0.4', :configuration => 'Debug'
+  pod 'Flipper-RSocket', '~> 1.0', :configuration => 'Debug'
```

Comment:
I wasn't sure what to do about version numbers for the transitive dependencies. If the FlipperKit pods change its dependencies to a different version number, we'll need to update these.

I exposed this problem in the cocoapods repo:
https://github.com/CocoaPods/CocoaPods/issues/9658#issuecomment-608260488

For the time being I can't find another solution to this.

Reply:
I guess the parameter to the `use_flipper!` method should take a single options hash and then apply defaults. Something like:

```ruby
def use_flipper!(versions = {})
  versions['Flipper'] ||= '~> 0.33.1'
  versions['DoubleConversion'] ||= '1.1.7'
  # etc

  pod 'Flipper', versions['Flipper'], :configuration => 'Debug'
  pod 'Flipper-DoubleConversion', versions['DoubleConversion'], :configuration => 'Debug'
  # etc
end
```

Reply:
@alloy I just made those changes, let me know if there is anything else you would like to see :)

Reply:
Ace, thanks! 👌